### PR TITLE
Add/gcp

### PR DESCRIPTION
**Pull request by theavege**
Please include to stdlib [GCP](https://github.com/Crashedmind/PlantUML-icons-GCP)

### CODE REVIEW

**Comment by arnaudroques** [comment]
Great!
Thanks for your contribution.
[It is working now](https://www.plantuml.com/plantuml/uml/fP5HIiGm48RVFSNOfmeBUm6HrUBwf8WzG0d9MCAcIPWaxlKT4wja2OkBVMYOvlk_JAQx40N5j9WhuETQMscIGhYTfTy-3Q-3MnPdxngRlEDJnErWN5BJFbaPjREXXRvWF3ewQ3klj2Oy2cEcjsGmdBMNTNgqixRO8iVeICnh7wLYC9CdlVKdzVXrzGrqUOKHwGEfupfZy8DrDt-JEPzTplzJTQyK9_aD0lQaqQepfce-VLrowfcNWSHxTRq2GO6-HTNu-gggD0z27f8FlnqLgDRK_xTlVPpoq94ssItRxZK2ENBn4MLubvsmuqHQp2S0)!

**Review by The-Lum — COMMENTED** [review]
Hello,

Here is a minor pseudo-similar remark as `k8s`:
- Could you replace "local URL" to "stdlib URL"?

_Maybe also rename the file to `example.puml`, but that is another request (This is an other story).. [we need to think a little more]_

Regards,
Th.

File: gcp/Networking.puml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@startuml 
+
+!include Common.puml
+
+!include Networking/all.puml
+!include GCPSimplified.puml
+
```
[the remark this answers is not in the document]
Comment (theavege) [reply gcp/Networking.puml:7]: I'm sorry. I read it late.

[the remark this answers is not in the document]
Comment (The-Lum) [reply gcp/Networking.puml:7]: @theavege:
- Do you need help making corrections?
- Are you going to do them?

Regards,
Th.

[the remark this answers is not in the document]
Comment (The-Lum) [reply gcp/Networking.puml:7]: @theavege: _[Apologie, but PlantUML team would like to deliver a version this week...]_
- Do you need help making corrections?
- Should I do it? _as you want: ask me..._

Regards,
Th.

**Pull request merged**